functions/Get-DbaDatabase.ps1

```diff
@@ -0,0 +1,162 @@
+﻿FUNCTION Get-DbaDatabase
+{
+<#
+.SYNOPSIS
+Gets a SQL database object for each database that is present in the target instance of SQL Server.
+
+.DESCRIPTION
+ The Get-SqlDatabase command gets a SQL database object for each database that is present in the target instance of
+ SQL Server. If the name of the database is provided, the command will return only this specific database object.
+	
+.PARAMETER SqlInstance
+SQL Server name or SMO object representing the SQL Server to connect to. This can be a collection and recieve pipeline input.
+
+.PARAMETER SqlCredential
+PSCredential object to connect as. If not specified, current Windows login will be used.
+
+.PARAMETER System
+Returns only system databases
+
+.PARAMETER User
+Returns only user databases
+	
+.PARAMETER State
+Returns only online databases
+
+.PARAMTER Owner
```

Comment:
Missing e breaks parameter help. Should this parameter really be named owner? I'd kind of expect to be allowed to ask for a specific owner, rather than "everything not owned by sa". Maybe something like "HasCustomOwner".